### DISCUSSION TITLE
Register .deso gTLD

### DISCUSSION THREAD

**Discussion by starykolarz** [deso-protocol/dips#166] [Medium CIPs]
Idea for discussion - register the ".deso" top level domain.

This would enable developers to place their products/apps in that domain after DeSo Foundation approval. 

Benefits:
 - DeSo Foundation approval would be a sign that app meets certain standards (which would need to be developed)
 - clear sign app is based on decentralized social, specifically on DeSo blockchain layer 1
 - no need for devs to add DeSo in every app name...

Drawbacks:
- it would require some kind of centralized body & cash to operate

Info here: https://newgtlds.icann.org/en/applicants/global-support/faqs/faqs-en

**Comment by tijno** [comment]
love the idea

but didnt the application window close in april 2012

i cant find any info on any new application windows opening

**Comment by starykolarz** [comment]
There were 3 windows up to now if I remember correctly - 2000, 2003, 2012.  It's not open currently, but there are discussions around the subject - that's why I proposed to keep this in mind:
https://circleid.com/posts/20210716-what-icann-should-do-now-to-help-future-applicants-of-new-gtlds
https://dig.watch/processes/new-gtld-program

ICANN is working on the next round - last document that is about the subject and quite specific I found is from 2019: https://mm.icann.org/pipermail/council/attachments/20190617/3ef07b78/Pre-EngagementCommsAttachment_AssumptionsPaper6.7.191-0001.pdf

DeSo Foundation is probably not ready for this now - but if it decides that .deso gTLD is a good way to promote Decentralized Social - preparations would have to start well ahead of starting of next round by ICANN.
